pulseaudio_dlna/pulseaudio.py

```diff
@@ -473,6 +475,9 @@ def update(self):
 
     def cleanup(self):
         for bridge in self.bridges:
+            if self.persist_stream and bridge.device.state == bridge.device.PLAYING:
+                logger.info('Stop silent stream on device "{}" ...'.format(bridge.device.label))
+                bridge.device.stop()
```

Comment:
You won't need that block. If the application terminates the HTTP stream collapses and the device will stop by itself. Normally what you are doing here is a very good thing, but in this case it will make the applications shutdown block until all stop commands return. This can take a while ...